x-pack/legacy/plugins/siem/public/components/timeline/search_or_filter/pick_events.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiHealth, EuiSuperSelect } from '@elastic/eui';
+import React, { memo } from 'react';
+import styled from 'styled-components';
+
+import { EventType } from '../../../store/timeline/model';
+import * as i18n from './translations';
+
+interface EventTypeOptionItem {
+  value: EventType;
+  inputDisplay: React.ReactElement;
+}
+
+const AllEuiHealth = styled(EuiHealth)`
+  margin-left: -2px;
+  svg {
+    stroke: #fff;
+    stroke-width: 1px;
+    stroke-linejoin: round;
+    z-index: 2;
+    width: 19px;
+    height: 19px;
+    margin-top: 1px;
+    z-index: 1;
```

Comment:
You have `z-index` duplicated as 2 and 1 here.

Reply:
yes correct, is too overlap the two health like the design below
![image](https://user-images.githubusercontent.com/189600/72371780-0bab5a80-36d3-11ea-8c9c-b2abece54e02.png)

Reply:
Oh I see what I did here